### ISSUE TITLE
broken links not caught

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/link-check#12]
https://github.com/iterative/dvc.org/pull/3275/checks passed even when that PR removed a page that was at the time linked from multiple places (links fixed in https://github.com/iterative/dvc.org/pull/3327).

Was this a link checker error?

**Comment by julieg18** [comment]
No, I don't think so. I'm pretty sure the link-check action that runs on prs only checks the links inside the edited files. Since it didn't run for the entire site, it didn't detect the broken links. But, @rogermparent, am I wrong?

**Comment by rogermparent** [comment]
@julieg18 is exactly right here, it specifically reads the git diff on PRs like the original script it replaced did.

However, this opens up an interesting question: would be it better just to go through the whole site and target that use case? It would catch situations like this and mean we maintain a lot less code. It would also make it easier to use something like `lychee` and drop us having to maintain a link check package altogether.

**Comment by julieg18** [comment]
> However, this opens up an interesting question: would be it better just to go through the whole site and target that use case?

Currently, we have around 40 broken links throughout the site. If we can get those cleared up and get a package that has a low chance of false negatives, I don't see a problem with having a check that checks the entire site on prs. But if we did it now, the check would probably be too noisy for every pr...

**Comment by casperdcl** [comment]
- check all links might take a long time (several minutes) each commit?
- we'd still probably want a way to run link-checks locally for convenience

**Comment by jorgeorpinel** [comment]
Yeah no need to change the behavior. But there is a site-wide link check process somewhere right? I keep forgetting where to see that. Would be ideal getting notified as well.

**Comment by jorgeorpinel** [comment]
OK so I guess this issue is unnecessary. We can work on https://github.com/iterative/dvc.org/issues/2486 instead for now. Thanks
